### PR TITLE
Dynamics of single-cell protein covariation during epithelial–mesenchymal transition

### PR DESCRIPTION
**Pull request by EnesSefaAyar**
QFeatures object available at https://zenodo.org/records/10696447

### CODE REVIEW

**Comment by EnesSefaAyar** [comment]
@lgatto I finalized the work on the new dataset and it is ready for review.

File: R/data.R [hunk 1]
```diff
@@ -2364,3 +2364,116 @@
...
+##' (`EpiToMesen.TGFB.nPoP_trial1_ProtByCellMatrix_NSThreshDART_medIntCrNorm_unimputed.csv`).
+##' The data were formated and added exactly as imputed data.
+##' 
+##' @source
+##' The data were downloaded from the
+##' [Slavov Lab](https://scp.slavovlab.net/Khan_et_al_2023) website via a
+##' shared Google Drive
+##' [folder](https://drive.google.com/drive/folders/1zCsRKWNQuAz5msxx0DfjDrIe6pUjqQmj).
+##' The raw data and the quantification data can also be found in the
+##' massIVE repository `MSV000092872`:
```
Comment (samgregoire) [inline R/data.R:2462]:
```suggestion
##' MassIVE repository `MSV000092872`:
```

File: inst/scripts/make-metadata.R [hunk 2]
```diff
@@ -601,6 +601,34 @@ meta <- list(
       ProteinsAvailable = TRUE,
       ContainsSingleCells = TRUE,
       Notes = NA_character_
+    ),
+    data.frame(
+      Title = "khan2023",
+      Description = paste0("Single-cell proteomics data ",
+                           "of 421 MCF-10A cells undergoing",
```
Comment (samgregoire) [inline inst/scripts/make-metadata.R:608]:
```suggestion
                           "of 421 MCF-10A cells undergoing ",
```

File: inst/extdata/metadata.csv [hunk 3]
```diff
@@ -21,3 +21,4 @@
...
 "21","woo2022_lung","Single-cell proteomics data from primary human lung cells.","3.18",NA,"TXT","ftp://massive.ucsd.edu/MSV000085937/",NA,"Homo sapiens",9606,TRUE,"MassIVE","Christophe Vanderaa <christophe.vanderaa@uclouvain.be>","QFeatures","Rda","scpdata/woo2022_lung.Rda",2022-05-18,5,"MaxQuant","LFQ",FALSE,TRUE,TRUE,TRUE,NA
 "22","gregoire2023_mixCTRL","Single-cell proteomics data from two monocyte cell lines","3.19",NA,"TXT","https://www.ebi.ac.uk/pride/archive/projects/PXD046211",NA,"Homo sapiens",9606,TRUE,"PRIDE","Samuel Gregoire <samuel.gregoire@uclouvain.be>","QFeatures","Rda","scpdata/gregoire2023_mixCTRL.Rda",2024-01-22,119,"Sage","TMT-16",TRUE,TRUE,TRUE,TRUE,NA
+"23","khan2023","Single-cell proteomics data of 421 MCF-10A cells undergoingEMT triggered by TGFβ","3.19",NA,"TXT","ftp://MSV000092872@massive.ucsd.edu/",NA,"Homo sapiens",9606,TRUE,"MassIVE","Enes Sefa Ayar <enes.ayar@uclouvain.be>","QFeatures","Rda","scpdata/khan2023.Rda",2023-12-21,47,"MaxQuant","TMTPro 16plex",TRUE,TRUE,TRUE,TRUE,NA
```
Comment (samgregoire) [inline inst/extdata/metadata.csv:24]: Missing space between "undergoing" and "EMT". See make-metadata.R

File: inst/scripts/make-data_khan2023.R [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+## 'annotation.csv' = the cell annotation (remainder operation needs to be used //16)
+## 'batch.csv' = the batch annotation 
+
+root <- "~/localdata/SCP/khan2023/002-singleCellDataGeneration/"
+ev <- read.delim(paste0(root, "ev_updated_NS.DIA.txt"))
+design <- read.csv(paste0(root, "annotation.csv"))
+batch <- read.csv(paste0(root, "batch.csv"))
+
+## Clean the sample metadata so that it meets the requirements for
+## `scp::readSCP`. The cell annotation and batch annotation are merge into a
```
Comment (samgregoire) [inline inst/scripts/make-data_khan2023.R:27]:
```suggestion
## `scp::readSCP`. The cell annotation and batch annotation are merged into a 
```

**Review by samgregoire — APPROVED** [review]

**Comment by lgatto** [comment]
You can provide Lori with this link to the update metadata file: 

https://github.com/UCLouvain-CBIO/scpdata/blob/master/inst/extdata/metadata.csv

**Pull request merged**